configCivcraft.yml

```diff
@@ -2836,11 +2836,11 @@ recipes:
     input:
       diamond ore:
         material: DIAMOND_ORE
-        amount: 20
+        amount: 32
     output:
       diamond:
         material: DIAMOND
-        amount: 24
+        amount: 64
```

Comment:
Wat??

You can't possibly expect people to get this utterly absurd amount of materials to make that factory for such utterly meager returns, it's just insane, please just fucking try playing the game it's actually painful. It would be really great if you just spoke to me in mumble or something to review the config it's so utterly broken I don't even know where to being, I mean ffs the best reinforcements are just stupid, they would never pay off to use it's hundreds of thousands of items for a stack. I can't help you balance this if you fight me on every single line out of some hypothetical balance philosophy that means nothing to players in the game.